kino-nextjs/app/lib/data.ts

```diff
@@ -1,5 +1,8 @@
 import { moviesList } from "../../public/mockMovies";
```

Comment:
This can be deleted? Or are you using the mockData somewhere?

Reply:
Yes it can. I just left it because i made a example in my last PR of rendering movies on all movies page.